03app_inriamuseum_showroom/03app_inriamuseum_showroom.c

```diff
@@ -75,10 +76,13 @@ void _ntw_receive_cb(uint8_t* buf, uint8_t bufLen);
 
 int main(void) {
     uint8_t txBuf[1];
-
+    uint8_t f_run_harry_potter;
```

Comment:
I don't understand how this would work. You are declaring variables locally to this function, then use it in another function. This will not work, you need to declare as part of the `app_vars` struct. Please reach out if this isn't clear, it's pretty fundamental